### PR TITLE
Fix #5 - Download image with background

### LINKED ISSUE #5: Download image with background
Like in context menu we have option to export image, but it download image without background, make the functionality that download image with background, also add another menu item for this.

### PR DESCRIPTION
**Pull request by antonioolf**
Fixed the issue #5

### CODE REVIEW

**Review by lablnet — CHANGES_REQUESTED** [review]
Thanks you so much for your contribution .

But pls make another button for that because we love to let user want download in both way if they want.
Look at here => https://github.com/lablnet/board/blob/master/index.html#L36
And  add button their Export with background, 

add second param to this function bool type, and say like withBg? 
the default value should be false, and on above btn click pass it true so it will download with background thanks.

**Comment by antonioolf** [comment]
Ok, i will do

**Comment by antonioolf** [comment]
@lablnet done!

File: js/app.js [hunk 1]
```diff
@@ -150,15 +150,17 @@ const ctx = board.getContext("2d");
...
+const downland = (withBg = false) => {
     const link = document.createElement('a');
     const locale = new Date().toLocaleString();
     const filename = `board${locale}.png`;
     link.download = filename;
 
-    ctx.globalCompositeOperation="destination-over";
-    ctx.fillStyle = localStorage.getItem("bcolor");
-    ctx.fillRect(0, 0, ctx.canvas.width, ctx.canvas.height);
+    if (withBg) {
```
Comment (lablnet) [inline js/app.js:159]: you should remove this if condition and pass the statements to `if (type == 'download-with-bg') {` here
and remove param withParam becausse it didnt need at all,

**Review by lablnet — CHANGES_REQUESTED** [review]

**Comment by antonioolf** [comment]
@lablnet this way?

**Comment by lablnet** [comment]
> @lablnet this way?

because we're not passing any param from menu to function so it will not work!

**Review by lablnet — APPROVED** [review]

**Comment by lablnet** [comment]
This `PR` is ready to be merge

**Comment by antonioolf** [comment]
Great @lablnet!

**Comment by lablnet** [comment]
> Great @lablnet!

Thanks you so much for contributing

**Pull request merged**